README.md

```diff
@@ -1,75 +1,69 @@
-# KintoHub {name}-example
+# KintoHub Django Example
 
 ## Overview
-TODO: Cool description / goal of what technology you are bringing onto the plate with what language. This should be sweet and concise.
 
-[Live Example](live/example/on/kintohub)
+Django is a high-level Python Web framework that encourages rapid development and clean, pragmatic design. It can be built easily for microservice on KintoHub!
+
+[Live Example](https://8caf94bb-c2e5-45cf-898d-ca5b19954cda.api.master.kintohub.com/django-example/hello/world/)
```

Comment:
Example does not work on KintoHub.

Reply:
tested on both Safari an Chrome, should work

Reply:
Works now.